### ISSUE TITLE
html-react-parser and dangerouslySetInnerHTML are both terrible

### ISSUE THREAD

**Issue by peterbe** [mdn/yari#5]
Unfortunately, there is good solution for rendering in HTML strings without a wrapper element. 

See https://github.com/facebook/react/issues/12014

If you do:
```jsx
let stuff = "<p>paragraph</p>";
<p dangerouslySetInnerHTML={{__html: stuff}}/>
```
you get...
```html
<p><p>paragraph</p></p>
```

[`html-react-parser`](https://github.com/remarkablemark/html-react-parser) is OKish but it adds 21KB to the bundle (8KB gzipped) which just too much at this early stage. I'd rather learn to live with a bunch of `<div>` tags everywhere. 

Basically, this issue is about keeping an eye on the above mentioned issue.

**Comment by peterbe** [comment]
Perhaps https://github.com/jonathantneal/react-dom-fragment?

**Comment by peterbe** [comment]
What's this? https://www.npmjs.com/package/html-to-react
